terraform/projects/app-ecs-albs-dev/main.tf

```diff
@@ -39,13 +39,25 @@ variable "project" {
   default     = "app-ecs-albs-dev"
 }
 
+data "terraform_remote_state" "infra_networking" {
+  backend = "s3"
+
+  config {
+    bucket = "${var.remote_state_bucket}"
+    key    = "infra-networking-modular.tfstate"
+    region = "${var.aws_region}"
+  }
+}
+
 module "app-ecs-albs" {
   source = "../../modules/app-ecs-albs/"
 
   aws_region          = "${var.aws_region}"
   stack_name          = "${var.stack_name}"
   remote_state_bucket = "${var.remote_state_bucket}"
   project             = "${var.project}"
+  zone_id             = "${data.terraform_remote_state.infra_networking.public_zone_id}"
+  subnets             = "${data.terraform_remote_state.infra_networking.public_subnets}"
```

Comment:
Optional: Wonder if we should always be explicit that these are `public_subnets` rather than just any `subnets`?

Reply:
I deliberately chopped `public_` off these names because I think that if app-ecs-albs is offering it as an input, then app-ecs-albs shouldn’t care what kind of subnet it is.

Reply:
I deliberately chopped `public_` off these names because I think that if app-ecs-albs is offering it as an input, then app-ecs-albs shouldn’t care what kind of subnet it is.